display/tab.go

```diff
@@ -128,7 +128,8 @@ func makeNewTab() *tab {
 
 		key := event.Key()
 		mod := event.Modifiers()
-		ru := event.Rune()
+		// ru := event.Rune()
+		cmd := config.TranslateKeyEvent(event)
```

Comment:
Since the whole `event` is being looked at, get rid of the `key` and `mod` variables, as well as the `ru` comment. Then fix the logic below to just look at `cmd`.